server/app/routes/api/v1/annotations/index.js

```diff
@@ -1,16 +1,23 @@
 'use strict';
```

Comment:
indeed, I didn't realize that when you commit to the same branch, it updates the pull request. So there are a number of features loaded into one pull request....